tests/unit/provision/mrack/test_hw.py

```diff
@@ -788,6 +789,35 @@ def test_zcrypt_mode(root_logger: Logger) -> None:
         }
 
 
+def test_iommu_is_supported(root_logger: Logger) -> None:
+
+    result = _CONSTRAINT_TRANSFORMERS['iommu.is_supported'](
+        _parse_iommu({"is-supported": True}), root_logger)
+
+    assert result.to_mrack() == {
+        'system': {
+            'key_value': {
+                '_key': 'VIRT_IOMMU',
+                '_op': '==',
+                '_value': '1'
+                }
+            }
+        }
+
+    result = _CONSTRAINT_TRANSFORMERS['iommu.is_supported'](
+        _parse_iommu({"is-supported": False}), root_logger)
+
+    assert result.to_mrack() == {
+        'system': {
+            'key_value': {
+                '_key': 'VIRT_IOMMU',
+                '_op': '==',
+                '_value': '0'
+                }
+            }
+        }
```

Comment:
how about this into a for cycle like `for value in True, False:` since that is the only thing different for these 2 sections? then at the `_value` field you can have `str(int(value))`

Reply:
@qiankehan Looks the related test function could be refined according to the suggestion above, e.g.
```python
def test_iommu_is_supported(root_logger: Logger) -> None:

    for value in True, False:
        result = _CONSTRAINT_TRANSFORMERS['iommu.is_supported'](
            _parse_iommu({"is-supported": value}), root_logger)

        assert result.to_mrack() == {
            'system': {
                'key_value': {
                    '_key': 'VIRT_IOMMU',
                    '_op': '==',
                    '_value': str(int(value)) 
                    }
                }
            }
```

Reply:
@qiankehan we would like to merge this, can you maybe improve this?

Reply:
Covered in 9deed7ef.